tnt-transfer-obj/src/main/java/org/dcsa/tnt/transferobjects/RelatedDocumentReferenceTO.java

```diff
@@ -0,0 +1,12 @@
+package org.dcsa.tnt.transferobjects;
```

Comment:
(Optional) This class can probably go. We don't care much about 2.2 compatibility and in 3.0 the fields are the same in all 3 events.

Reply:
They are not the same on the 3.0 TNT documentation on swaggerhub.

Reply:
https://app.swaggerhub.com/apis/dcsaorg/DCSA_TNT/3.0.0-Beta-1#/Events/get_v3_events

All three event types use relateddocumentreferences with a "type" and a "value"